docs/guides/compute-daemons/readme.md

```diff
@@ -0,0 +1,90 @@
+---
+authors: nate.roiger <nate.roiger@hpe.com>
+categories: setup
+---
+
+# Compute Daemons
+
+Rabbit software requires two daemons be installed and run on each compute node. Each daemon shares similar build, package, and installation processes described below.
+
+- The [***Client Mount***](https://github.com/HewlettPackard/dws/tree/master/mount-daemon) daemon provides the support for mounting Rabbit hosted file systems on compute nodes.
+- The [***Data Movement***](https://github.com/NearNodeFlash/nnf-dm/tree/master/daemons/compute) daemon supports creating, monitoring, and managing data movement (copy-offload) operations
+
+
+## Building from source
+
+Each daemon can be built in their respective repsitories using the `build-daemon` make target. Go version >= 1.19 must be installed to perform local build.
+
+## RPM Package
+
+Each daemon is packaged as part of the commit process. Source and Binary RPMs are available.
```

Comment:
Do you mean build process in Github Actions? If so, might be worth calling that out.